templates/configmap-rudder-server.yaml

```diff
@@ -10,3 +10,7 @@ data:
 {{ .Files.Get "rudder-config.toml" | indent 4}}
   google-application-credentials.json: |-
 {{.Files.Get "rudder-google-application-credentials.json" | indent 4}}
+  {{- if .Values.backend.controlPlaneJSON}}
```

Comment:
Instead of using a variable, can we check if the file is present and not empty in helm repo?

Reply:
Use template function?

Reply:
Do you mean not having the controlPlaneJSON variable and check if workspaceConfig file is present    then enable configFromFIile true ?

Reply:
yeah

Reply:
Actually, having a separate config is not that bad also. It will be more explicit way. Never mind my comments. Approving the PR